packages/block-editor/README.md

```diff
@@ -423,6 +423,14 @@ _Returns_
 
 -   `string`: Gradient value.
 
+<a name="ImageEditingProvider" href="#ImageEditingProvider">#</a> **ImageEditingProvider**
+
+Undocumented declaration.
+
+<a name="ImageEditor" href="#ImageEditor">#</a> **ImageEditor**
+
+Undocumented declaration.
+
```

Comment:
@ellatrix do you have an opinion on making the image editing tools part of the block-editor package? I think that the public API could be simplified to not need the provider as a separate component. I could do that here or mark it as experimental and do that in a follow-up if you think that would be better.

Reply:
#23155 was asking for the image editor to be exported for other blocks too.

Reply:
I went ahead and marked it as experimental in 28ba223